src/GenAPI/Microsoft.DotNet.GenAPI/SyntaxGeneratorExtensions.cs

```diff
@@ -92,5 +95,20 @@ private static ConstructorInitializerSyntax GenerateBaseConstructorInitializer(I
 
             return constructorInitializer;
         }
+
+        private static BaseListSyntax? GetBaseTypeList(this SyntaxGenerator syntaxGenerator,
```

Comment:
Please add a comment to describe the method's purpose. I assume its for filtering.

Reply:
added `Gets the list of base class and interfaces for a given symbol <see cref="INamedTypeSymbol"/>.`